static/_redirects

```diff
@@ -24,4 +24,5 @@
 /istio /blog/2018/08/07/istio-10-cilium 301
 /blog/2020/11/10/ebpf-future-of-network /blog/2020/11/10/ebpf-future-of-networking 301
 
-/* https://cilium.netlify.app/:splat 200
\ No newline at end of file
+https://blog.cilium.io/* /blog/:splat 200!
```

Comment:
What does the `!` at the end do?

Reply:
it forces redirect. for example there is an existing page at https://blog.cilium.io/, but it redirects to https://blog.cilium.io/blog anyways